R/IDateTime.R

```diff
@@ -240,6 +240,22 @@ rep.ITime = function (x, ...)
   class(y) = "ITime"   # unlass and rep could feasibly not copy, hence use class<- not setattr()
   y
 }
+                           
+round.ITime <- function(x, digits = c("hours", "minutes"), ...) 
```

Comment:
I still think the signature here should use `units=` as the argument name if possible. That should also solve the Travis issue we're seeing IINM

Reply:
Ahh, I see. Actually `round.IDate` is already using `digits=` as an argument, and `units=` is the undocumented object.

For consistency with `round.IDate`, I'm OK with `round.ITime` using `digits=`. Then we just have to document the `units` argument & should be OK

Reply:
exactly. I noticed it when doing the documentation :)